src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -146,6 +146,24 @@ export default class ReviewCollapsibleChapter extends React.Component {
             });
             const title = page.reviewTitle || page.title || '';
 
+            const hasVisibleFields =
+              pageSchema &&
+              Object.entries(pageSchema.properties)
+                .map(entry => entry[0])
+                .filter(propName =>
```

Comment:
I think we could be more descriptive here about `entry` and do some destructuring rather than have a hardcoded index which can be confusing at first glance.

```js
              Object.entries(pageSchema.properties)
                .map(([propName, ]) => propName)
                .filter(propName =>
```

Or we could even remove the map altogether:


```js
              Object.entries(pageSchema.properties).filter(([propName]) =>
                showReviewField(
                  propName,
                  pageSchema,
                  pageUiSchema,
                  form.data,
                )
```